db/db_compaction_test.cc

```diff
@@ -206,6 +206,7 @@ Options DeletionTriggerOptions(Options options) {
       options.target_file_size_base * options.target_file_size_multiplier;
   options.max_bytes_for_level_multiplier = 2;
   options.disable_auto_compactions = false;
+  options.compaction_options_universal.max_size_amplification_percent = 100;
```

Comment:
I changed this from the default (200) because "CompactionDeletionTrigger" was in the [190, 200) range and relying on size-ratio compaction, which just so happened to pick through to the last sorted run. Size-amp is more reliable as it's guaranteed to pick through the last sorted run.